### ISSUE TITLE
third option for diploid confusing

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#1940]
I wasn't sure what the bottom row referred to here:

![bottom row?](https://user-images.githubusercontent.com/7359272/59331290-80588980-8ceb-11e9-91df-238075d89144.jpg)

It seemed to be the same as the second row?

**Comment by ValWood** [comment]
When used -  it created  the suc1 example diploid:

![other option](https://user-images.githubusercontent.com/7359272/59331382-b990f980-8ceb-11e9-9a10-5398c2146c3b.jpg)

**Comment by ValWood** [comment]
So, the difference is that  the 2nd option gives WT product level and the third gives not assayed.

It might be better not to have the 3rd option.  Usually if the WT copy was not altered it should be WT expression. If bby any chance they did some more engineering to alter the expression, we can adjust that  via edit details?

**Comment by kimrutherford** [comment]
I don't understand this.  Looks like a major bug.

Which session is it?

**Comment by ValWood** [comment]
I saw it int eh ones I was fixing....
https://github.com/pombase/curation/issues/2418

so I think ed12ab2f3f0456f6 which was the last one I fixed for sure

**Comment by kimrutherford** [comment]
I don't think it's that session.

It's very alarming that you selected cut7delta but ended up with a suc1 diploid!

**Comment by ValWood** [comment]
I didn't. Sorry , I took the screenshots when annotating different  diploids. The problem isn't that major!

**Comment by mah11** [comment]
I think I see what's happening here. The third option is for creating a diploid that is neither homozygous for the selected allele nor the heterozygote formed of the selected allele plus wild type. So it shows the selected allele plus any alleles of the same gene entered in the session that are neither wild type nor the first allele, and offers a pull-down selector if there's more than one option.

Examples: 

I added some silly alleles for testing here: 
![Screen Shot 2019-06-12 at 10 22 17 AM](https://user-images.githubusercontent.com/5196259/59342755-49da3900-8d02-11e9-8a77-9518c187ff87.png)

Then if I select exo1∆, these options come up:
![Screen Shot 2019-06-12 at 10 22 41 AM](https://user-images.githubusercontent.com/5196259/59342767-5068b080-8d02-11e9-92d8-09a62ad1604f.png)


whereas if I pick one of the other alleles first, I get these options:
![Screen Shot 2019-06-12 at 10 23 01 AM](https://user-images.githubusercontent.com/5196259/59342775-5494ce00-8d02-11e9-8260-8fe9a3d65296.png)


It's all working as expected and we absolutely DO need the third option whenever there are alleles other than wild type or deletion for a gene.

Finally, the reason Val's cut7 and suc1 examples have those funny options like "cut7∆ cut7+/cut7-(heterozygous diploid)" is that Canto is offering up the instances where diploids were shoehorned in as "alleles of type other" - it can't tell them apart from the actual single alleles. IOW, Canto thinks "cut7+/cut7-(heterozygous diploid)" is one allele of cut7, so it offers to combine it with the selected cut7∆.

**Comment by mah11** [comment]
tl;dr: there is no problem with Canto here.

**Comment by ValWood** [comment]
> Finally, the reason Val's cut7 and suc1 examples have those funny options like "cut7∆ cut7+/cut7-(heterozygous diploid)" is that Canto is offering up the instances where diploids were shoehorned in as "alleles of type other" - it can't tell them apart from the actual single alleles. 

of course. This issue should go away once  https://github.com/pombase/curation/issues/2418
finsihed. So this  ticket needs to be closed before  diploids are unleashed..

**Comment by ValWood** [comment]
Hang on, I'm a bit confused. This isn't an  genotypes existing in the database that's showing, it' the cut7 diploid genotyoes that I just created in this session. (otherwise why would it only show up this particular cut7 allele and not another exisitng ones like "cut7-24 and cut7-21"?


So in this option, we don't want to serve  up any "diploids"

**Comment by kimrutherford** [comment]
> So in this option, we don't want to serve up any "diploids"

I should only show the single allele in the diploid dialog.  Could you send me a session where this is happening?

**Comment by ValWood** [comment]
It was in curs/894e3c36003ab516
but I no longer see it.

![cut7](https://user-images.githubusercontent.com/7359272/59349054-a0e70a80-8d10-11e9-90fc-d7560e0fcd54.jpg)

 so Midori was correct, but that implies that in option 3, where we want to select non-deletion or WT alleles this isn't the options we get.

**Comment by mah11** [comment]
> but that implies that in option 3, where we want to select non-deletion or WT alleles this isn't the options we get.

Well, no. The reason you don't see a third option in the latest screenshot is that that session no longer has any cut7 alleles other than wt and deletion. If you add any other cut7 allele, it'll appear in the third option.

**Comment by ValWood** [comment]
Durr, yes I see, this was the original allele in *this*  session, so now it's deleted I don't see it. 

But now I know why the 3rd option is there. It isn't so clear.  This option only appears if you have added other alleles of cut7.  A user might try to create a diploid  without creating the other allele and in this case, they won't be able to.

**Comment by kimrutherford** [comment]
Is there anything to do here?

**Comment by ValWood** [comment]
nope, sorry!
